src/lua/oltp_multi_value_insert.lua

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sysbench
+
+-- ----------------------------------------------------------------------
+-- Multi-rows in a single Insert OLTP benchmark
+-- ----------------------------------------------------------------------
+
+require("oltp_common")
+
+sysbench.cmdline.commands.prepare = {
+   function ()
+      if (not sysbench.opt.auto_inc) then
+         -- Create empty tables on prepare when --auto-inc is off, since IDs
+         -- generated on prepare may collide later with values generated by
+         -- sysbench.rand.unique()
+         sysbench.opt.table_size=0
+      end
+
+      cmd_create()
+      --cmd_prepare()
+   end,
+   sysbench.cmdline.PARALLEL_COMMAND
+}
+
+function prepare_statements()
```

Comment:
needed?

Reply:
Yes this is needed, oltp_common.sh expects this function to be defined.